consensus/poet/core/tests/test_consensus/test_poet_config_view.py

```diff
@@ -36,7 +36,7 @@ def test_key_block_claim_limit(self, mock_config_view):
         # pylint: disable=protected-access
         self.assertEqual(
             poet_config_view.key_block_claim_limit,
-            PoetConfigView._DEFAULT_KEY_CLAIM_LIMIT_)
+            PoetConfigView._KEY_BLOCK_CLAIM_LIMIT_)
```

Comment:
Not sure that I like this private access of these values.  If someone changes these constants, nothing breaks, and there's a lower barrier to think about why the values are set the way they are.  

In terms of code coverage, since you're testing the defaulting of the values, you're really not providing much other than that you've linked the correct constant.